services/certificate-api/src/utilities/cord.js

```diff
@@ -0,0 +1,105 @@
+// src/utilities/cord.js
+// CORD-related functions.
+
+import * as cord from '@cord.network/sdk'
+
+import { CORD_CHAIN_ADDRESS } from '../../config/index.js'
+
+// First, connect to the chain.
+await cord.init({ address: CORD_CHAIN_ADDRESS })
+
+// Retrieve the identities used to create schemas and credentials.
+// TODO: `Alice` works, but a newly created account doesn't. Why?
```

Comment:
a few developer accounts are funded by default. If you are creating a new account, transfer some credits from one of the developer accounts.

Reply:
I did do that, and it threw an 'insufficient funds to pay transaction fees' error.

Reply:
We are building a tool to request for funds through a portal. Once we host it, will share more details, for now, just msg the account ID which need some tokens in CORD discord.